tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Immutable from 'immutable';
+import * as helloWorldActionCreators from '../actions/helloWorldActionCreators';
+
+function select(state) {
+  // Which part of the Redux global state does our component want to receive as props?
+  return { $$helloWorldStore: state.$$helloWorldStore };
+}
+
+class HelloWorld extends React.Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    $$helloWorldStore: PropTypes.instanceOf(Immutable.Map).isRequired,
+  }
+
+  render() {
+    const { dispatch, $$helloWorldStore } = this.props;
+    const actions = bindActionCreators(helloWorldActionCreators, dispatch);
+    return (
+      <HelloWorldWidget $$helloWorldStore={$$helloWorldStore} actions={actions} />
```

Comment:
How about this @robwise:

Let's show the alternate, short version as a comment:


```javascript
  
  // NOTE: alternate syntax which is a bit more concise.
  // <HelloWorldWidget {...{$$helloWorldStore, actions}} />
  // Inner part is an object where key is variable name.
  // Outer part is how React JSX will use the destructuring to make the Object be the same as using = for properties.
  return (
      <HelloWorldWidget $$helloWorldStore={$$helloWorldStore} actions={actions} />
```